bash/utils.sh

```diff
@@ -261,6 +263,7 @@ function fgdb_export_partial {
             -lco GEOMETRY_NAME=Shape\
             -nln ${nln}\
             -nlt ${geomtype}\
+            -t_srs ${target_projection}\
             "$@"
         rm -f ${filename}.gdb.zip
```

Comment:
`ogrinfo -al -so ${filename}.gdb` seems really nice to have in product build logs anytime we generate a geodabase, it's a lot of good info for not a lot of lines

maybe running it right before or after line 268 with the `rm -f ${filename}.gdb.zip`?

Reply:
@damonmcc, see output with the geodabase summary [here](https://github.com/NYCPlanning/data-engineering/actions/runs/7424876744/job/20205383668#step:14:24)